packages/cli/cli.js

```diff
@@ -39,6 +39,8 @@ async function load (moduleName) {
 }
 
 const ensureCommand = async ({ output, help }) => {
+  globalThis.platformatic = { executable: 'platformatic', resolvedServicesFolder: RESOLVED_SERVICES_DIRNAME }
```

Comment:
Why is this needed?

Reply:
Rather than trying to autodetect the executable (`watt` vs `plt`), I chose the executable to be explicit. This will be used in the runtime error handlers.